### PR TITLE
PRJ-613 Adding basic handling of drawRenderedImage() for BufferedImages

### PR DESCRIPTION
**Pull request by tomb50**
Adding basic handling of drawRenderedImage() for BufferedImages

### CODE REVIEW

**Comment by tomb50** [comment]
https://youtrack.jetbrains.com/issue/PRJ-613

Original it appears just to be an issue with nullability, but then I noticed that the method was not really implemented, no ServerDrawEvent being fired.

Following the SunGraphics2D impl. drawRenderedImage() for BufferedImage can be treated the same as drawImage - this is what I need for my use case so I havent tried to implement any of the other general cases.

File: projector-awt/src/main/kotlin/org/jetbrains/projector/awt/image/PGraphics2D.kt [hunk 1]
```diff
@@ -186,8 +186,19 @@ class PGraphics2D private constructor(
     paintShape(AwtPaintType.DRAW, s)
   }
 
-  override fun drawRenderedImage(img: RenderedImage, xform: AffineTransform) {
-    paintPlain { drawRenderedImage() }
+
+  override fun drawRenderedImage(img: RenderedImage, xform: AffineTransform?) {
+    // xform nullability is required for compatibility, so provide a default (identity) transformation
+    val xFormOrDefault = xform?: AffineTransform()
```
Comment (SerVB) [inline projector-awt/src/main/kotlin/org/jetbrains/projector/awt/image/PGraphics2D.kt:192]:
```suggestion
  override fun drawRenderedImage(img: RenderedImage, xform: AffineTransform?) {
    // xform nullability is required for compatibility, so provide a default (identity) transformation
    val xFormOrDefault = xform ?: AffineTransform()
```

**Review by SerVB — APPROVED** [review]
Cool, thanks! There is only a minor code style change.

**Comment by tomb50** [comment]
Awesome, thanks!

**Pull request merged**